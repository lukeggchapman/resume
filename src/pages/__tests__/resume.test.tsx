import React from 'react'
import { shallow } from 'enzyme'

import ResumePage from '../resume'

jest.mock('components/SEO', () => 'SEO')
jest.mock('components/ResumePDF', () => 'ResumePDF')

describe('ResumePage', () => {
  const render = () => shallow(<ResumePage />)

  it('renders correctly', () => {
    expect(render()).toMatchSnapshot()
  })
})
