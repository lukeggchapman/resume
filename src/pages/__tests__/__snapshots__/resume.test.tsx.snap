// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResumePage renders correctly 1`] = `
<Fragment>
  <SEO
    title="Résumé"
  />
  <ResumePDF />
</Fragment>
`;
